### ISSUE TITLE
`serve` command not serving prerendered SSR or static export content

### ISSUE THREAD

**Issue by thescientist13** [ProjectEvergreen/greenwood#1072]
## Type of Change
Bug

## Summary
After upgrade to [**v0.28.0-alpha.2**](https://github.com/ProjectEvergreen/greenwood/releases/tag/v0.28.0-alpha.2), a project that was originally using `prerender: true` and an SSR route now shows a 404 page when serving the content.  
https://github.com/AnalogStudiosRI/www.tuesdaystunes.tv/pull/110
![Screen Shot 2023-02-20 at 9 38 07 AM](https://user-images.githubusercontent.com/895923/220140571-8c1dbdff-cd85-4c92-acc7-7c603792efdb.png)

It should be noted the _index.html_ IS correctly generated, just not being served as expected to the browser.

## Details
Doing a little digging and I think what happened is now with the fix to #1059 in place, both static server and hybrid server are not running together anymore for just static content (a good thing!).  However, now that no static server is running, [this line](https://github.com/ProjectEvergreen/greenwood/blob/v0.28.0-alpha.2/packages/cli/src/lifecycles/serve.js#LL165C7-L165C106) basically precludes _prerendered_ SSR content from being returned now.
```js
if (isSPA || (matchingRoute && !matchingRoute.isSSR) || url.pathname.split('.').pop() === 'html') {
  ...
}
```

So basically need to make sure static and hybrid server are working together here:
1. Make sure static server only handles static (or prerendered SSR) routes
1. Let all non static routes get handled by hybrid server
